docs/v3.x/guides/custom-admin.md

```diff
@@ -29,7 +29,19 @@ Let's eject this file to be able to customize it.
 
 In this new file, paste the current [HomePage container](https://github.com/strapi/strapi/tree/master/packages/strapi-admin/admin/src/containers/HomePage/index.js) code.
 
-To run your application, you will have to run the `yarn develop --watch-admin` command.
+To run your application, run the following command:
+
+yarn
+
+```bash
```

Comment:
Still need tabs here

Reply:
sorry I dont get it. What do you mean by tab?

Reply:
Thanks, updated